docker-compose.yml

```diff
@@ -145,10 +137,8 @@ services:
       dockerfile: ./packages/notification/Dockerfile
     restart: unless-stopped
     environment:
```

Comment:
As webhooks is externally facing, I think that webhooks still needs this

Reply:
@euanmillar You must have mixed the service names. As I don't see any changes in `webhooks`. Rather these are the changes for `notification`

Reply:
Ah yes, sorry "Git files changed" makes it look like these belong in webhooks.  My bad